crates/fs/src/fs.rs

```diff
@@ -239,8 +241,16 @@ impl Fs for RealFs {
         } else {
             symlink_metadata
         };
+
+        #[cfg(unix)]
+        let inode = metadata.ino();
+
+        // todo(windows)
```

Comment:
Could you change this to `todo!(windows)`?

Reply:
I changed it